codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -264,13 +274,24 @@ class FluentClientGenerator(
             val errorType = symbolProvider.symbolForOperationError(operation)
 
             // Have to use fully-qualified result here or else it could conflict with an op named Result
```

Comment:
Is this comment still relevant? I don't see the result it's referring to.

Reply:
Good catch! Removed the comment in 4982446.